common/header.html

```diff
@@ -17,7 +17,8 @@
           const isException = category &&
           settings.exceptions.split("|").includes(category.name);
 
-          if(/^\/c\//.test(path)) {
+          const categoryPathChecker = new RegExp("^" + Discourse.BaseUri + "\\/c\\/");
```

Comment:
`Discourse.BaseUri` is not escaped. It should not be a problem, but it's better to be safe than sorry.

```suggestion
          const escapedBase = Discourse.BaseUri.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
          const categoryPathChecker = new RegExp("^" + escapedBase + "\\/c\\/");
```

Reply:
@udan11 can you merge the suggestion and approve+merge the PR if it's ready to go? 🙂